submissions/AaLin-Git/html-css-popup/index.html

```diff
@@ -0,0 +1,223 @@
+                    </a>
+                  </li>
+                  <li class="popupLink">
+                    <a class="menuPopupItem" href="#">
+                      <img class="menuPopupIcon" src="./img/venus.svg" alt="" />
+                      <span class="textItem">Venus</span>
+                    </a>
+                  </li>
+                  <li class="popupLink">
+                    <a class="menuPopupItem" href="#">
+                      <img class="menuPopupIcon" src="./img/earth.svg" alt="" />
+                      <span class="textItem">Earth</span>
+                    </a>
+                  </li>
+                  <li class="popupLink">
+                    <a class="menuPopupItem" href="#">
+                      <img class="menuPopupIcon" src="./img/moon.svg" alt="" />
+                      <span class="textItem">Moon</span>
+                    </a>
+                  </li>
+                  <li class="popupLink">
+                    <a class="menuPopupItem" href="#">
+                      <img class="menuPopupIcon" src="./img/mars.svg" alt="" />
+                      <span class="textItem">Mars</span>
+                    </a>
+                  </li>
+                  <li class="popupLink">
+                    <a class="menuPopupItem" href="#">
+                      <img
+                        class="menuPopupIcon"
+                        src="./img/jupiter.svg"
+                        alt=""
+                      />
+                      <span class="textItem">Jupiter</span>
+                    </a>
+                  </li>
+                  <li class="popupLink">
+                    <a class="menuPopupItem" href="#">
+                      <img
+                        class="menuPopupIcon"
+                        src="./img/saturn.svg"
+                        alt=""
+                      />
+                      <span class="textItem">Saturn</span>
+                    </a>
+                  </li>
+                  <li class="popupLink">
+                    <a class="menuPopupItem" href="#">
+                      <img
+                        class="menuPopupIcon"
+                        src="./img/uranus.svg"
+                        alt=""
+                      />
+                      <span class="textItem">Uranus</span>
+                    </a>
+                  </li>
+                  <li class="popupLink">
+                    <a class="menuPopupItem" href="#">
+                      <img
+                        class="menuPopupIcon"
+                        src="./img/neptune.svg"
+                        alt=""
+                      />
+                      <span class="textItem">Neptune</span>
+                    </a>
+                  </li>
+                </ul>
+                <input type="checkbox" id="menuInputSub" class="input" />
+                <label class="labelSub more" for="menuInputSub">More </label>
```

Comment:
Logically this button must be hidden when the second part of the popup is open.

Reply:
I changed button more to less, when it is pressed :)

Reply:
From a UX perspective, it's not ok because users see a list of links then they see the button and after this button, they see the second part of the popup. What if you have a couple of parts of links inside the popup, you will add a button before each of them or only before the last one?

Reply:
It depends on the purpose. If this is a header menu, then it is better to add one button before the last part to be able to collapse the extended menu to the main one. And if this is a menu filter for an online store, for example, then I would make a popup for each section.